teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
+    @specialization = specialization
+    super(age, name, parent_permission: parent_permission)
+  end
+
+  def can_use_servicies?
+    puts true
+  end
```

Comment:
- [ ] Instead of printing `true` to the console, kindly return it! so every time you use this method it returns a boolean. 🤓 🔧 

According to the requirements:
![image](https://github.com/HFG43/OOP-school-library/assets/77069011/45ccfecc-0103-4b5b-b4dc-402bf991121f)

Reply:
@anagudelogu Thanks for your feedback Andres! Requested changes applied